waymo_raw_data_to_argoverse.py

```diff
@@ -45,6 +52,14 @@
     'ring_side_right', # 5, 'SIDE_RIGHT'
 ]
 
+LABEL_TYPES = [
```

Comment:
could you add an in-line comment explaining what the indices here denote? which ones are argo, and which one waymo labels?

Reply:
Addressed in [this commit](https://github.com/johnwlambert/waymo_to_argoverse/pull/2/commits/8931614c713c462a1efbbc0f06a366d44a28df25)